### PR TITLE
Fix #8

### LINKED ISSUE #8: The twig snippet doesn't work anymore
I tried to use the snippet of the README, but this ended in the following error:
`An exception has been thrown during the rendering of a template ("Oneup\Contao\SentryBundle\Twig\ContaoSentryTwigRuntime::sentryLastEventIdFilter(): Return value must be of type ?string, Sentry\EventId returned")`

We fixed it locally via explicitly casting the LastEventId to a string, e.g.:
```php
    public function sentryLastEventIdFilter(): ?string
    {
        return (string)SentrySdk::getCurrentHub()->getLastEventId();
    }
```
A better solution would of course be to check if the LastEventId is null and only if it is not then casting it to a string.
I'm running on PHP 8.3 with Contao 4.13.35

### PR DESCRIPTION
**Pull request by f-schnabel**

### CODE REVIEW

File: src/Twig/ContaoSentryTwigRuntime.php [hunk 1]
```diff
@@ -10,7 +10,11 @@ class ContaoSentryTwigRuntime
 {
     public function sentryLastEventIdFilter(): ?string
     {
-        return SentrySdk::getCurrentHub()->getLastEventId();
+        $lastEventId = SentrySdk::getCurrentHub()->getLastEventId();
+        if ($lastEventId === null) {
```
Comment (bytehead) [inline src/Twig/ContaoSentryTwigRuntime.php:15]: Missing new line
```suggestion

        if ($lastEventId === null) {
```

**Review by bytehead — CHANGES_REQUESTED** [review]

File: src/Twig/ContaoSentryTwigRuntime.php [hunk 2]
```diff
@@ -10,7 +10,12 @@ class ContaoSentryTwigRuntime
 {
     public function sentryLastEventIdFilter(): ?string
     {
-        return SentrySdk::getCurrentHub()->getLastEventId();
+        $lastEventId = SentrySdk::getCurrentHub()->getLastEventId();
+
+        if ($lastEventId === null) {
+            return null;
+        }
+        return (string) $lastEventId;
```
Comment (bytehead) [inline src/Twig/ContaoSentryTwigRuntime.php:19]: Same here, sorry 😅 
```suggestion

        return (string) $lastEventId;
```

**Review by bytehead — CHANGES_REQUESTED** [review]

File: src/Twig/ContaoSentryTwigRuntime.php (hunk 2 above)
Comment (f-schnabel) [reply src/Twig/ContaoSentryTwigRuntime.php:19]: no problem :)

**Review by bytehead — APPROVED** [review]

**Comment by bytehead** [comment]
Thank you! ❤️

**Pull request merged**